src/content/docs/using-new-relic/welcome-new-relic/get-started/get-started-full-stack-observability.mdx

```diff
@@ -41,11 +46,9 @@ Our platform goes beyond simple monitoring by offering you observability: data i
   A comparison between simple monitoring and observability.
 </figcaption>
 
-## Get your data into New Relic [#integrations]
+## Get your data into New Relic with our quickstarts [#quickstarts]
 
-[New Relic I/O](https://developer.newrelic.com/instant-observability/) is a rich catalog of open source quickstarts - out-of-the-box bundles of integrations, dashboards, and alerts. 
-
-Each quickstart is created by observability experts around the world, vetted by New Relic, and ready for you to install with one click. Leverage community expertise and get more value out of your telemetry data with New Relic I/O, your hub for instant observability.
+[New Relic I/O](https://developer.newrelic.com/instant-observability/) is a rich catalog of open source quickstarts - out-of-the-box bundles of integrations, dashboards, and alerts. Popular technologies such as Node.js, Python, and Ruby have full-featured quickstarts, while others contain a mixture of tools. Each quickstart is created by observability experts around the world, vetted by New Relic, and ready for you to install with one click. Leverage community expertise and get more value out of your telemetry data with New Relic I/O, your hub for instant observability.
 
```

Comment:
I personally am not a fan of em-dashes (they don't always render correctly and are fiddly to create). And also, "out of the box" seems like a 1990s throwaway. We don't sell our software in boxes! :-) Maybe consider rewording to "New Relic I/O is a rich catalog of open-source quickstarts that automatically include integrations, dashboards, and alerts for you to use immediately."

Reply:
Gotcha!